### PR TITLE
Sorting by relationship properties for union and interface fields

### PR DESCRIPTION
**Pull request by darrellwarde**
# Description

Allows for the sorting by relationship property values on interface/union relationship fields.

# Issue

#414 

# Checklist

The following requirements should have been met (depending on the changes in the branch):

- [ ] Documentation has been updated
- [x] TCK tests have been updated
- [x] Integration tests have been updated
- [ ] Example applications have been updated
- [x] New files have copyright header
- [x] CLA (https://neo4j.com/developer/cla/) has been signed

### CODE REVIEW

File: packages/graphql/src/translate/connection/create-connection-and-params.ts [hunk 1]
```diff
@@ -254,6 +254,13 @@ function createConnectionAndParams({
 
         const subqueryCypher = ["CALL {", subqueries.join("\nUNION\n"), "}"];
 
+        if (sortInput && sortInput.length) {
+            const sort = sortInput.map((s) =>
+                [...Object.entries(s.edge || []).map(([f, direction]) => `edge.${f} ${direction}`)].join(", ")
```
Comment (angrykoala) [inline packages/graphql/src/translate/connection/create-connection-and-params.ts:259]: I'm not sure why destructuring and creating a new array here. Wouldn't `Object.entries().map` already return an array? 
```suggestion
                Object.entries(s.edge || []).map(([f, direction]) => `edge.${f} ${direction}`).join(", ")
```

File: packages/ogm/src/generate.test.ts [hunk 2]
```diff
@@ -18,13 +18,13 @@
...
-import generate from "./generate";
-import { OGM } from "./index";
 import * as fs from "fs";
 import * as path from "path";
+import generate from "./generate";
+import { OGM } from "./index";
 
 describe("generate", () => {
-    let filesToDelete: string[] = [];
+    const filesToDelete: string[] = [];
```
Comment (angrykoala) [inline packages/ogm/src/generate.test.ts:27]: :clap:

**Review by angrykoala — DISMISSED** [review]

File: packages/graphql/src/translate/connection/create-connection-and-params.ts (hunk 1 above)
Comment (darrellwarde) [reply packages/graphql/src/translate/connection/create-connection-and-params.ts:259]: Ahahaha, my mistake, maybe guilty of a little copy/paste from another part of the code where there is two array entries... 🙄

**Comment by darrellwarde** [comment]
Merged based off the previous approval of @angrykoala, which was dismissed by merging his single suggestion.

**Pull request merged**